packages/terra-overlay/tests/wdio/overlay-spec.js

```diff
@@ -0,0 +1,31 @@
+/* global before, browser, Terra */
```

Comment:
Do you know why the text appears differently on the various screenshots?

Reply:
It is because of the padding value being passed in the test.